src/utils/domUtils.ts

```diff
@@ -1,23 +1,3 @@
-let size: number | undefined;
-
-export function getScrollbarSize(): number {
-  if (size === undefined) {
-    const scrollDiv = document.createElement('div');
-
-    scrollDiv.style.position = 'absolute';
-    scrollDiv.style.top = '-9999px';
-    scrollDiv.style.width = '50px';
-    scrollDiv.style.height = '50px';
-    scrollDiv.style.overflow = 'scroll';
-
-    document.body.appendChild(scrollDiv);
-    size = scrollDiv.offsetWidth - scrollDiv.clientWidth;
-    document.body.removeChild(scrollDiv);
```

Comment:
We won't manually trigger layout/reflow in the middle of rendering now 👌 👌 👌

Reply:
👏

Reply:
@nstepien 
Could you explain a little bit why we needed this originally? Is it for the scenario when the scroll bar appears and we need to get the new demension?

Reply:
We needed it to know what the scrollbar size was so we could correctly calculate the actual vertical/horizontal viewport.